doc/changelog.rst

```diff
@@ -1,6 +1,18 @@
 Changelog
 =========
 
+.. rubric:: 1.1
+
+- Use direction prediction for the measurement equation, rather than FFT and
```

Comment:
direction prediction == direct FT?

Reply:
Thanks, I meant to write "direct prediction". Fixed.